src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,38 @@
+package duke.task;
+
+public abstract class Task {
+    private String description;
+    private TaskType taskType;
```

Comment:
nitpicking here a bit, but don't you think typeOfTask would sound better?

Reply:
I agree! 😅